examples/internal/integration/client_test.go

```diff
@@ -2,13 +2,18 @@ package integration_test
 
 import (
 	"context"
+	"fmt"
 	"testing"
 
+	"github.com/rogpeppe/fastuuid"
+
 	"github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/clients/abe"
 	"github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/clients/echo"
 	"github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/clients/unannotatedecho"
```

Comment:
Sorry, I meant to put these all in the same import group. We generally use 2 import groups
```suggestion
	"github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/clients/abe"
	"github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/clients/echo"
	"github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/clients/unannotatedecho"
	"github.com/rogpeppe/fastuuid"
```